src/services/azureFunctionsService.ts

```diff
@@ -77,7 +77,14 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 				// start the create azure function project flow
 				await azureFunctionApi.createFunction({});
 			}
-			return;
+			if (await azureFunctionUtils.getHostFiles()) {
+				// once project is created then start sql binding flow
+				projectFile = await azureFunctionUtils.getAzureFunctionProject();
+			} else {
+				// if user cancels prompt show warning
+				vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened);
```

Comment:
So if the user clicks the learn more button doesn't that mean that they're going to get the same notification twice in a row? That doesn't seem like a great experience, if they choose learn more we should either

1. End the flow there
2. Show the error message with the buttons again (for the case where they want to learn more but also do want to make a new project)

(I think 2 would be my preference, @dzsquared thoughts?)

Reply:
I personally like the first one as showing the same message would possibly create confusion, and they should be able to understand that in order to run is to create an azure function stated in the docs and in the error message they already saw. So they would have to re-run the flow or create an azure function project separately.

Reply:
I don't think it'd be that confusing - it's pretty standard to have docs links not take you away from the page/dialog you're on. But it's a minor detail so I'm fine leaving it like this too.